setup.py

```diff
@@ -43,6 +29,24 @@ def get_version(rel_path):
     return "{}.{}.{}".format(major_version, minor_version, patch_version)
 
 
+def get_install_requires():
+    install_requires = [
+        'neo4j',
+        'praatio~=5.0',
+        'textgrid',
+        'conch_sounds',
+        'librosa',
+        'influxdb',
+        'tqdm',
+        'requests',
+        'scipy~=1.12.0',
+        'setuptools' # only added to work with praatio
+    ]
+    if os.name == 'nt':
```

Comment:
I've only seen this check as `sys.platform == 'win32'`, but maybe they're identical, but would probably stick to sys.platform since that's the standard

Reply:
Oh, I guess you can do something like this: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-and-requirements, that should clean it up

Reply:
Done!